src/test/java/org/savantbuild/dep/workflow/process/WorkflowTest.java

```diff
@@ -44,6 +45,61 @@
  * @author Brian Pontarelli
  */
 public class WorkflowTest extends BaseUnitTest {
+  @Test
+  public void fetchSource_publish_no_process_exists() throws Exception {
```

Comment:
I don't think this name is correct. The process exists fine, but the source file doesn't exist. Maybe call it `fetchSource_publish_source_file_does_not_exist` or something.

Reply:
Updated